### ISSUE TITLE
Make it explicity in the relevant pages that entrants keep ownership of their submissions

### ISSUE THREAD

**Issue by mkalam-alami** [alakajam-team/alakajam#620]
Proposal: https://gist.github.com/jlegeny/31c4c04d9e1641044037f2241fea77ce

Add something like that to the privacy policy page, plus a couple shortcuts to those paragraphs from the jam rules pages.

Sources: https://github.com/alakajam-team/alakajam/tree/master/server/docs/article-data

**Comment by jlegeny** [comment]
I agree that the third part is quite superfluous, I think this is mostly aimed at hackathons. It might be useful to add for website submissions but it's pointless for the jams themselves. Maybe it could just have a friendlier message saying that "you give us the right to use the images you upload"

**Comment by mkalam-alami** [comment]
We have something similar in the F.A.Q. we can reuse: https://alakajam.com/article/docs/faq
